### ISSUE TITLE
Mura's Bootstrap 3 Dropdowns are not keyboard accessible

### ISSUE THREAD

**Issue by dgibson666** [blueriver/MuraCMS#1451]
Since the code governing this is in Mura Core, I decided to file the bug here rather than the bootstrap3 theme.

The "hack" that is used to make the top level links active links work (rather than only toggle the menu) is what is breaking keyboard nav. In requirements/mura/content/contentrenderer.cfc in dspPrimaryNav():

&lt;!--- hack or issue with bootstrap that breaks link with kids ---&gt;
&lt;cfif arguments.aHasKidsClass eq 'dropdown-toggle' and arguments.aHasKidsAttributes eq 'role="button" data-toggle="dropdown" data-target="##"'&gt;
	&lt;cfset arguments.aHasKidsAttributes=''&gt;
&lt;/cfif>

So in my navbar.cfm call to dspPrimaryNav(), I removed the data-target part...

, aHasKidsAttributes='role="button" data-toggle="dropdown"'

This results in the parent nav item (toggling the dropdown) is no longer clickable as an active link, but the subnav is fully functional and keyboard accessible. While data-target="##" (or any value it seems) is present, you cannot open the dropdown with the keyboard at all. I am not sure why the Mura hack causes keyboard inaccessibility, but there's got to be some way to fix this.
